metrics-otlp/README.md

```diff
@@ -0,0 +1,149 @@
+# Install an OTLP-based metrics collector 
+
+## Overview
+
+The following instructions demonstrate how to install [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector)s on a Kyma cluster using the official [Helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-collector) with the goal to collect and ship workload metrics to an OTLP endpoint. For a fully Prometheus-based approach, have a look at the [Prometheus](./../prometheus/README.md) tutorial instead.
+
+The setup brings an OpenTelemetry Collector Deployment acting as gateway, to which all the cluster-wide metrics should be ingested. Then, the gateway enriches the metrics with missing resource attributes and ships them to a target backend.
+
+Furthermore, the setup brings an OpenTelemetry Collector DaemonSet acting as agent running on each node. Here, node-specific metrics relevant for your workload are determined. This is also where an annotation-based workload scraper is running, which scrapes all containers running on the related node.
+## Architecture
+
+[Architecture](./assets/overview.drawio.svg)
+
+## Prerequisites
+
+- Kyma Open Source 2.10.x or higher
+- kubectl version 1.22.x or higher
+- Helm 3.x
+
+## Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_NS="{namespace}"
+    ```
+
+1. If you haven't created a Namespace yet, do it now and enable Istio injection with the following label:
+    ```bash
+    kubectl create namespace $KYMA_NS
+    kubectl label namespace $KYMA_NS istio-injection=enabled
+    ```
+
+1. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+    helm repo update
+    ```
+
+## Install the gateway
```

Comment:
Why does the headline say "install", but all related instructions talk about "deploy"...?
 
```suggestion
## Deploy the gateway
```